tests/integration/tier0/non-destructive/duplicate-pkgs/main.fmf

```diff
@@ -9,6 +9,7 @@ enabled: false
 
 /duplicate-pkgs:
     link: https://issues.redhat.com/browse/RHELC-1070
+    enabled: false
```

Comment:
Why is this test no longer enabled? I don't see any change in the jira link.

Reply:
TL;DR we try to work with the smallest sample when working with enablement and comparisons.
It's kind of awkward, but we have decided previously, that we will try to keep the plan/test enablement conditioning simple.
The comma separated values, can be interpreted as `a or b` whereas `and` cannot be shortened.
So we rather go with `==` instead of `!=` when comparing the distro context and (ideally) the shorter list of affected systems.
Just a line below, there is:
```
adjust+:
    - enabled: true
      when: distro == centos-7, oracle-7
```
Which is shorter and imho more readable, than:
```
adjust+:
    - enabled: true
      when: distro != centos-8 or distro != oracle-8 or distro != oracle-9 or distro != alma-8 or distro != alma-9 or distro != rocky-8 or distro != rocky-9
```
or 
```
- enabled: false
  when: distro == centos-8, oracle-8, oracle-9, alma-8, alma-9, rocky-8, rocky-9
```
Does that make sense?